### PR TITLE
Set JEMALLOC_PAGE_SIZE env variable

### PR DESCRIPTION
**Pull request by MichaReiser**
An attempt to fix https://github.com/astral-sh/ruff/issues/13506

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,12 +6,16 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a5bf44b1aa0adaf6d9d20f86162b34f7c593bfedabc51239953e446aefc8ce18
+  fn: uv-{{ version }}.tar.gz
+  url: https://github.com/astral-sh/ruff/releases/download/{{ version }}/source.tar.gz
+  sha256: 4f3c5d0acbefb5792ad82a903137f18e8498726dca3b678bdc3b8f8129948341
```
Comment (zanieb) [inline recipe/meta.yaml:11]: Hm?

Comment (MichaReiser) [reply recipe/meta.yaml]: I tried to align the recipe with uv's but looks complicated :laughing:

**Comment by MichaReiser** [comment]
I'm not sure how to test this change. The logs for aarch and powerpc show that the environment variable is set. But the checks create no artifacts that we could download ;(

**Comment by bcaddy** [comment]
I'm happy to test it since I raised the original issue. How do I get a binary with the new build?

**Comment by zanieb** [comment]
I don't think conda-forge publishes downloadable artifacts for these jobs; maybe @hmaarrfk knows?

**Comment by hmaarrfk** [comment]
see: https://conda-forge.org/docs/maintainer/conda_forge_yml/#azure
```
store_build_artifacts
```

**Review by zanieb — APPROVED** [review]
I think this probably can't hurt

**Comment by zanieb** [comment]
Will test in https://github.com/conda-forge/ruff-feedstock/pull/225

**Comment by bcaddy** [comment]
I can also confirm that Ruff from Conda-forge now works on my aarch64 machine. Thanks all.

**Pull request merged**